shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -321,6 +301,32 @@ private void ensureValidAndroidVersion(int minSdkVersion) {
         }
       };
 
+  private static MotionEvent toMotionEvent(
+      float density, PlatformViewsChannel.PlatformViewTouch touch) {
+    PointerProperties[] pointerProperties =
+        parsePointerPropertiesList(touch.rawPointerPropertiesList)
+            .toArray(new PointerProperties[touch.pointerCount]);
+    PointerCoords[] pointerCoords =
+        parsePointerCoordsList(touch.rawPointerCoords, density)
+            .toArray(new PointerCoords[touch.pointerCount]);
+
+    return MotionEvent.obtain(
+        touch.downTime.longValue(),
+        touch.eventTime.longValue(),
+        touch.action,
+        touch.pointerCount,
+        pointerProperties,
+        pointerCoords,
+        touch.metaState,
+        touch.buttonState,
+        touch.xPrecision,
+        touch.yPrecision,
+        touch.deviceId,
+        touch.edgeFlags,
+        touch.source,
+        touch.flags);
```

Comment:
It looks like we don't pass any of these values :/ https://github.com/flutter/flutter/pull/60320#discussion_r448067059